src/libs/antares/study/parts/hydro/series.h

```diff
@@ -27,6 +27,7 @@
 #ifndef __ANTARES_LIBS_STUDY_PARTS_HYDRO_TIMESERIES_H__
 #define __ANTARES_LIBS_STUDY_PARTS_HYDRO_TIMESERIES_H__
 
+#include <include/antares/series/series.h>
```

Comment:
We should not have the "include" folder, here, see comment on the définition of the include dir in CMakeLists.txt